src/DynamoPackages/PackageLoader.cs

```diff
@@ -69,9 +69,13 @@ public IEnumerable<IExtension> RequestedExtensions
         public IEnumerable<Package> LocalPackages { get { return localPackages; } }
 
         private readonly List<string> packagesDirectories = new List<string>();
+
+        // Returns the default package directory where new packages will be installed
+        // The default package directory is currently the second entry in the list
+        // The first entry is the standard library
         public string DefaultPackagesDirectory
         {
-            get { return packagesDirectories[0]; }
+            get { return packagesDirectories[1]; }
```

Comment:
We need to do something about this property but I think that should be part of @mjkkirschner's PR.

Reply:
@sm6srw what were you thinking 😉 ?